modules/compose/templates/docker-compose.yml.erb

```diff
@@ -83,9 +83,9 @@ services:
 # DOCKER REGISTRY
 ########################
   registry:
-    image: registry:2.5.0
+    image: <%= ENV["IMAGE_REGISTRY"] %>
     env_file:
-      - '<%= scope.lookupvar('compose::codenvy_folder') -%>/config/registry/registry.env'
+      - '/codenvy/instance/config/registry/registry.env'
```

Comment:
By hard coding this AND removing the sed commands, we lose the ability to generate two docker-compose.yml files. If we want to generate docker-compose-container.yml and docker-compose-host.yml at the same time, we need to use sed to change the env file.s